app/views/reports/_auditing_report.html.haml

```diff
@@ -22,12 +22,12 @@
   .modal-content
     = form_tag reports_path, remote: true, class: "form-horizontal", role: "form" do
       .modal-header
+        %h4.modal-title.text-center
+          = t(:reports)[:auditing_report]
         %button.close{type: 'button', data: {dismiss: 'modal'}}
```

Comment:
The modal "x" close buttons were changed with Bootstrap 4. Could you update these to the new version?
```
%button.close{ type: 'button', data: { dismiss: 'modal' }, aria: { label: t('actions.close') } }
  %span{ aria: { hidden: 'true' } } &times;
```

Reply:
Buttons have been updated.